internal/cmd/cmds.go

```diff
@@ -8,9 +8,10 @@ import (
 )
 
 type DiceDBCmd struct {
-	RequestID uint32
-	Cmd       string
-	Args      []string
+	RequestID        uint32
+	Cmd              string
+	Args             []string
+	PreProcessingReq bool
```

Comment:
Let's add a comment here explaining what this is used for.

Reply:
Removed from DiceDBCmds commands and moved to the storeOp.